hpi-schul-cloud/tldraw-client
# BC-6593-Correct type display in upload dialog

# Description

QA note: Change tested on branch:
https://bc-6593-file-type-display-in-upload-dialog.brb.dbildungscloud.dev/tldraw?roomName=6601803664fced815ed99b7a
No issues found. See screen in jira ticket. 



## Links to Tickets or other pull requests

- https://ticketsystem.dbildungscloud.de/browse/BC-6593

## Changes



## Datasecurity



## Deployment



## New Repos, NPM pakages or vendor scripts



## Screenshots of UI changes



## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
